<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomPage</class>
 <widget class="QWidget" name="RoomPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(39, 44, 54);
}


QWidget#films_widget{
	border-radius: 10px;
	background-color: rgba(37, 55, 67, 90);
}

QLineEdit{
	min-height: 45px;
	border-radius: 20px;
	background-color: #FFFFFF;
	padding-left: 20px;
	color: rgb(140, 140, 140);
}

QLineEdit:hover{
	border: 2px solid rgb(139, 142, 139);
}

QPushButton{
	min-width: 80px;
	min-height: 45px;
	border-radius: 20px;
	background-color: rgb(44, 66, 81);
	color: #FFFFFF;
}

QPushButton:hover{
	border: 2px solid rgb(255, 255, 255);
}

QCheckBox{
	font-size: 10px;
	color: #FFFFFF;
}

QLabel{
	color: rgb(95, 94, 108);
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="buttons_frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="search_le">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	color: rgb(28, 43, 52);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Filmin AdÄ±</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="search_btn">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;add film&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Ara</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QScrollArea" name="films_sa">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="films_widget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>638</width>
          <height>425</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
